logprep/util/configuration.py

```diff
@@ -839,15 +836,6 @@ def reload(self) -> None:
         errors: List[Exception] = []
         try:
             new_config = Configuration.from_sources(self.config_paths)
-            if self._config_failure:
-                logger.info("Config refresh recovered from failing source")
-            self._config_failure = False
-            if new_config == self:
-                logger.info(
-                    "Configuration version didn't change. Continue running with current version."
-                )
-                self._set_config_refresh_interval(new_config.config_refresh_interval)
-                return
```

Comment:
We [keep this block in the new architecture](https://github.com/fkie-cad/Logprep/pull/882/files#diff-6c9f91348fe111796648b32ee3de8debd0be48ae56053dcfdd1f17d731730b91R842), but we delete it in the old one. Is there a reason for the discrepancy?

I wonder about the benefits of this conditional. Do we save CPU resources by shortcutting the case where the configs are equal? Do you have any insight on this?

Reply:
I believe it was added to save resources.
Furthermore, the manager did only reload if the version changed **and** `new_config == self` was true.
Now the manager reloads on all version changes so I removed it.

I can change it in the ng version too. I was barely touching the ng version.